cypress/tests/ImageEmbed.spec.ts

```diff
@@ -66,7 +66,7 @@ describe("ImageEmbed", () => {
           typeIntoEmbedField("caption", "Caption text");
           typeIntoEmbedField("altText", "Alt text");
           assertDocHtml(
-            `<imageembed type="imageEmbed" has-errors="false"><div class="ProsemirrorEmbed__imageEmbed-caption"><p>Caption text</p></div><div class="ProsemirrorEmbed__imageEmbed-altText"><p>Alt text</p></div><embed-imageembed-usesrc class="ProsemirrorEmbed__imageEmbed-useSrc" fields="{&quot;value&quot;:false}"></embed-imageembed-usesrc></imageembed><p>First paragraph</p><p>Second paragraph</p>`
+            `<imageembed type="imageEmbed" has-errors="false"><div class="ProsemirrorEmbed__imageEmbed-altText"><p>Alt text</p></div><div class="ProsemirrorEmbed__imageEmbed-caption"><p>Caption text</p></div><embed-imageembed-usesrc class="ProsemirrorEmbed__imageEmbed-useSrc" fields="{&quot;value&quot;:false}"></embed-imageembed-usesrc></imageembed><p>First paragraph</p><p>Second paragraph</p>`
```

Comment:
Sadly necessary, as ordering now matters for our fields – and so we've changed our markup, which will always contain them in alphabetical order . It shouldn't matter for our consumers, which will have other ways of getting hold of element data.